codespell_lib/_codespell.py

```diff
@@ -407,14 +407,22 @@ def parse_options(args):
     config = configparser.ConfigParser()
 
     # Read toml before other config files.
+    toml_files_errors = list()
+    if os.path.isfile('pyproject.toml'):
+        toml_files_errors.append(('pyproject.toml', False))
     if options.toml:
+        toml_files_errors.append((options.toml, True))
+    for toml_file, raise_error in toml_files_errors:
         try:
             import tomli
         except Exception as exc:
-            raise ImportError(
-                f'tomli is required to read pyproject.toml but could not be '
-                f'imported, got: {exc}') from None
-        with open(options.toml, 'rb') as f:
+            if raise_error:
+                raise ImportError(
+                    f'tomli is required to read pyproject.toml but could not '
+                    f'be imported, got: {exc}') from None
+            else:
+                continue
```

Comment:
Do we not have a verbose log option where we could write something at that level?

Reply:
We could, but I'm not sure we need to. Someone expecting it to "just work" who doesn't have `tomli` might be confused for a second, then they'll presumably try `--toml <whatever>` and see an error about import.